### PR TITLE
add missing include in FEDataManager.cpp

### PR DESCRIPTION
**Pull request by boyceg**
### IBAMR Pull Request Checklist
- [ ] Does the test suite pass?
- [x] Was clang-format run?
- [x] Does this change include a bug fix or new functionality? If so, a new test
      or tests should be added. New tests should run quickly (less than a minute
      in release mode). If possible, an older test should gain a new option so
      that we do not need to compile more test executables.
- [x] Did you (if your account has permission to do so) set relevant labels on
      GitHub for the pull request?

### CODE REVIEW

**Comment by boyceg** [comment]
@drwells looks like another IWYU casualty.

File: ibtk/src/lagrangian/FEDataManager.cpp [hunk 1]
```diff
@@ -18,6 +18,8 @@
 #include "ibtk/FECache.h"
 #include "ibtk/FEDataManager.h"
 #include "ibtk/IBTK_CHKERRQ.h"
+#include "ibtk/IndexUtilities.h"
+#include "ibtk/JacobianCalculator.h"
```
Comment (drwells) [inline ibtk/src/lagrangian/FEDataManager.cpp]: I need to change these over to `<` and `>` at some point...

**Comment by drwells** [comment]
Its a header - lets merge without me running the test suite.

File: ibtk/src/lagrangian/FEDataManager.cpp (hunk 1 above)
Comment (boyceg) [reply ibtk/src/lagrangian/FEDataManager.cpp]: I always thought that the "rule" was to use angle brackets for system headers, quotes for non-system headers.

Comment (drwells) [reply ibtk/src/lagrangian/FEDataManager.cpp]: The only difference is that using `"`s also includes the current directory in the preprocessor path. Ultimately its not that big of a deal but I prefer to use the slightly stricter pathing rule.

**Comment by drwells** [comment]
@boyceg its not a big deal but could we keep all the bullet points, even the irrelevant ones? It is easier to tell at a glance what has been done if we use the same list.

**Comment by boyceg** [comment]
Sure thing. I was thinking that we should only include the relevant ones, to make sure to capture whether all relevant items have been done.

**Pull request merged**